cmd/src/batch_validate.go

```diff
@@ -60,9 +60,7 @@ Examples:
 		out := output.NewOutput(flagSet.Output(), output.OutputOpts{Verbose: *verbose})
 		ui := &ui.TUI{Out: out}
 		svc := service.New(&service.Opts{
-			Client:           cfg.apiClient(apiFlags, flagSet.Output()),
-			AllowUnsupported: allowUnsupported,
-			AllowIgnored:     allowIgnored,
+			Client: cfg.apiClient(apiFlags, flagSet.Output()),
```

Comment:
those flags were specific to a single method on service, so they aren't set here anymore to avoid confusing people thinking they're always mandatory to set (myself mostly)